wfl/Models/FoodOrderStatusDto.cs

```diff
@@ -0,0 +1,42 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Web;
+
+namespace Symphono.Wfl.Models
+{
+    public class FoodOrderStatusDto
+    {
+        public string Status { get; set; }
```

Comment:
I think we should consider moving this logic out of the DTO and into the business entity.  It comes down to who is responsible for what.  A DTO is simply a property bag of data that is passed around.  The business entity owns itself and all mutations to that entity.  I would expect this logic to live within that entity.